src/js/components/award/SummaryBar.jsx

```diff
@@ -0,0 +1,93 @@
+/**
+ * SummaryBar.jsx
+ * Created by Emily Gullo 01/20/2017
+ **/
+
+import React from 'react';
+import * as Icons from '../sharedComponents/icons/Icons';
+
+const propTypes = {
+    selectedAward: React.PropTypes.object,
+    getStatus: React.PropTypes.func,
+    awardStatus: React.PropTypes.string
+};
+
+export default class SummaryBar extends React.Component {
+
+    componentWillReceiveProps(nextProps) {
+        if (this.props.awardStatus === "" || this.props.awardStatus !== nextProps.awardStatus) {
+            if (this.props.selectedAward) {
+                this.props.getStatus(this.props.selectedAward);
+            }
+        }
+    }
+
+    render() {
+        let summaryTitle = null;
+        let awardId = null;
+        let parentAwardId = null;
+        let status = null;
+        let parentId = null;
+        if (this.props.selectedAward) {
+            const award = this.props.selectedAward;
+            if (award.parent_award) {
+                parentId = award.parent_award;
+            }
+            else if (!award.parent_award && award.type_description !== "Definitive Contract") {
```

Comment:
Comparing against the `type` code (listing can be found in `dataMapping/search/awardTypes.js`) is safer than comparing against the `type_description` string as it is less likely to change.